<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnuQuiz</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(255, 255, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>-10</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AnuQuiz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createQuizButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(58, 84, 255)
</string>
    </property>
    <property name="text">
     <string>Create Quiz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answerQuizButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(58, 84, 255)
</string>
    </property>
    <property name="text">
     <string>Answer Quiz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 0, 4)
</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
